samples/sqltransport/store-and-forward/sample.md

```diff
@@ -0,0 +1,101 @@
+---
+title: SQL Server transport store-and-forward
+summary: How to add store-and-forward functionality for external-facing endpoints
+reviewed: 2016-03-21
+tags:
+- SQL Server
+- Store-and-forward
+related:
+- nservicebus/outbox
+- nservicebus/sqlserver
+redirects:
+- samples/sqltransport-outbox-store-and-forward
+- samples/outbox/store-and-forward
+---
+
+
+## Prerequisites
+
+ 1. Ensure an instance SQL Server Express installed and accessible as `.\SQLEXPRESS`. Create two databases: `sender` and `receiver`.
+
+
+## Running the project
+
+ 1. Start the Sender project (right-click on the project, select the `Debug > Start new instance` option).
+ 1. Start the Receiver project. 
+ 1. In the Sender's console is the text `Press <enter> to send a message` when the app is ready.
+ 1. Hit `<enter>`.
+
+
+## Verifying that the sample works correctly
+
+ 1. The Receiver displays information that an order was submitted.
+ 1. The Sender displays information that the order was accepted.
+ 1. Hit `<enter>` in the Receiver console to shut it down.
+ 1. Go to the SQL Server Management Studio and delete the `receiver` database.
+ 1. Hit `<enter>` again in the Sender console
+ 1. Notice the retry mechanism kicks in after a 10 seconds delay. It attempts to retry sending the message to the destination database every 10 seconds.
```

Comment:
Or "Notice the retry mechanism kicks in after a 10 seconds and retries sending the message to the destination database."